integration/e2emimir/client.go

```diff
@@ -128,6 +129,37 @@ func (c *Client) Push(timeseries []prompb.TimeSeries) (*http.Response, error) {
 	return res, nil
 }
 
+// OTLPPush the input timeseries to the remote endpoint in OTLP format
+func (c *Client) OTLPPush(timeseries []prompb.TimeSeries) (*http.Response, error) {
```

Comment:
Nit, but I'd call this `PushOLTP`, as in "push _this format_". But it's very opinionated.